src/wallet/wallet.cpp

```diff
@@ -2295,6 +2295,12 @@ bool CWalletTx::InMempool() const
 }
 
 bool CWalletTx::IsTrusted(interfaces::Chain::Lock& locked_chain) const
+{
+    std::set<uint256> s;
+    return IsTrusted(locked_chain, s);
+}
+
+bool CWalletTx::IsTrusted(interfaces::Chain::Lock& locked_chain, std::set<uint256>& trustedParents) const
```

Comment:
nit, snake case - header too.

Reply:
Did we change styles?

Reply:
In 2014: 2887bffcfdc

Reply:
We settled on a variable naming style in 2017: 47d84414662